src/screens/rewards/components/RewardsPendingEarnings.tsx

```diff
@@ -23,29 +23,31 @@ export const RewardsPendingEarnings: React.FC<Props> = ({
   const nextAirdropIsInThePast = isPast(dayOfNextDistribution);
 
   return (
-    <RewardsSectionCard>
-      <Columns>
-        <Stack space="12px" alignHorizontal="left">
-          <Text color="labelTertiary" size="15pt" weight="semibold">
-            {i18n.t(i18n.l.rewards.pending_earnings)}
-          </Text>
-          <Text color="label" size="22pt" weight="heavy">
-            {`$${pendingEarningsUsd}`}
-          </Text>
-        </Stack>
-        <Stack space="12px" alignHorizontal="right">
-          <Text color="labelTertiary" size="15pt" weight="semibold">
-            {nextAirdropIsInThePast
+    <Box paddingBottom="36px">
```

Comment:
Nit: the date calculations above here could pry be memoized

Reply:
I think it might be only adding memoization overhead, the screen has barely any state, and the calculation will always run when the sheet is opened. I think it might be not worth it.

If I notice in subsequent PRs that indeed the component re-renders a couple times, I will add a memo there, until then I think it doesn't make sense.